### PR TITLE
Added two patches to 1-21

### PR DESCRIPTION
**Pull request by kschumy**
*Description of changes:*
* Added extend-sa-token-if-audience-is-apiserver-1.patch and Parse-ipv6-address-before-comparison-10773.patch to 1-21 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes/kubernetes/1-21/patches/0021-EKS-PATCH-Parse-ipv6-address-before-comparison-10773.patch [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+diff --git a/pkg/kubelet/nodestatus/setters.go b/pkg/kubelet/nodestatus/setters.go
+index 52f12178b66..a31cfbd2970 100644
+--- a/pkg/kubelet/nodestatus/setters.go
++++ b/pkg/kubelet/nodestatus/setters.go
+@@ -125,8 +125,10 @@ func NodeAddress(nodeIPs []net.IP, // typically Kubelet.nodeIPs
+ 
+ 				nodeIPTypes := make(map[v1.NodeAddressType]bool)
+ 				for _, nodeAddress := range cloudNodeAddresses {
+-					if nodeAddress.Address == nodeIP.String() {
++					if netutils.ParseIPSloppy(nodeAddress.Address).Equal(nodeIP) {
```
Comment (jaxesn) [inline projects/kubernetes/kubernetes/1-21/patches/0021-EKS-PATCH-Parse-ipv6-address-before-comparison-10773.patch:28]: Does parseipsloppy exists yet in 1-21/1-22?  Im not really sure, but I thought that was added after they bumped to golang 1.17 which changed the parsing rules.  The BR patch does it slightly differently, https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/kubernetes-1.21/0002-IPv6-node-address-before-comparing-with-node-I.patch#L29

**Comment by kschumy** [comment]
And so the Checksum battle begins again. There was a merge conflict with the updated, so I dropped that commit and am waiting for the job to fail. I'll update this PR with the new ones when they're available.

**Comment by kschumy** [comment]
/hold

Adding another patch

**Comment by kschumy** [comment]
I'm going to fix the patch not applying. I think I know what happened

**Comment by kschumy** [comment]
/unhold

not adding the patch at this time

**Comment by kschumy** [comment]
/hold 

Removing all the 1.22 stuff, as its in https://github.com/aws/eks-distro/pull/727

**Comment by kschumy** [comment]
/unhold

This PR should be done, unless checksum ruin everything again

**Review by jaxesn — APPROVED** [review]
/lgtm

**Pull request merged**